File: Python/Ex37.py
numeros = []

x = int(input('Digite a quantidade de valores: '))

while x<=0 or x>20:
    print(f"Erro! Digite um valor entre 1 e 20")
    x = int(input("Digite novamente: "))

for i in range(0,x,1): 
    n = int(input('Digite os números: '))
    numeros.append(n)


cons = int(input('Digite o número que deseja consultar: '))

valor=0

for i in range(0,20,1): 
    if cons==numeros[i]: 
     print(f"O número está na posição: {i} do vetor")
    else:
     valor=valor+1
    if valor==x:
     print('Valor não encontrado!')

File: C/EX37.c
#include<stdio.h>
#include<stdlib.h>

int main(int argc, char const *argv[])
{

int i,n,q,c,valor;
valor=0;

int *numeros;

do
{
system("cls");


printf("\nDigite a quantidade de valores: ");
scanf("%i", &q);


while(q>20 || q<0)
{
printf("Erro!\nFora da faixa possivel de valores. Digite novamente: ");
scanf("%i",&q);
}

numeros = (int*)malloc(q*sizeof(int));

printf("Digite os numeros:\n");
for(i=0; i<q; i++)
{
    scanf("%i", &numeros[i]);
}

printf("Digite um numero para consulta: ");
scanf("%i", &c);


for(i=0; i<q; i++)
{ 

   if (c==numeros[i])
   { 
       printf("O Numero %i se encontra na Posicao %i do vetor",c,i);
   }

   else 
   {
       valor++; 
   }

   if(valor==q)
   {
       printf("Valor nao encontrado!");
       valor=0; 
   }
}
printf("\nDigite 2 para sair, ou qualquer numero para nova consulta: ");
    scanf("%i",&n); 
}

while(n !=2);

return 0;
}